### PR TITLE
Add documentation for now v2 deployment

### PR DESCRIPTION
**Pull request by diverent2**
V1 is now offically deprecated and can no longer be used in new projects.
Add new documentation for V2

### Before submitting the PR, please make sure you do the following
- [X] Related to https://github.com/sveltejs/sapper-template/issues/102
- [X] This message body should clearly illustrate what problems it solves. If there are related issues, remember to reference them.
- [X] Ideally, include a test that fails without this PR but passes with it. PRs will only be merged once they pass CI. (Remember to `npm run lint`!)
### Tests
-  [X] Run the tests tests with `npm test` or `yarn test`)

### CODE REVIEW

**Comment by thgh** [comment]
With the latest version of now-sapper, you don't need `.handler` => just `export default app` (same for express and polka)

File: site/content/docs/11-deploying.md [hunk 1]
```diff
@@ -7,9 +7,57 @@ Sapper apps run anywhere that supports Node 8 or higher.
...
+```js
+polka() //same with Express
+  .use(
+    [...]
+  )
+```
+
+with
+```js
+const app = polka //same with Express
```
Comment (Vehmloewff) [inline site/content/docs/11-deploying.md]: Should there be parentheses here?

```suggestion
const app = polka() //same with Express
```

**Comment by diverent2** [comment]
@thgh I've updated to docs accordingly :)  
Thanks for your feedback!

**Comment by diverent2** [comment]
*ping* :)

**Comment by ryanditjia** [comment]
> With the latest version of now-sapper, you don't need `.handler` => just `export default app` (same for express and polka)

Perhaps the README in the repo could use an update? Polka still says `export default app.handler` 😄

**Comment by brancusi** [comment]
Works great thank you

**Comment by Conduitry** [comment]
Since this is such a desired feature, I think it's probably acceptable to have a link to direct people to using a third party solution (`now-sapper`) for this - but the official docs shouldn't take on the responsibility of explaining how to use it. We don't want people to come here if they're having trouble with it, and we don't want to have to worry about keeping the explanation in sync with now-sapper's real docs.

We should just have a link to `now-sapper` and its documentation, and mention that it is an unofficial project.

**Comment by benmccann** [comment]
@diverent2 will you have a chance to consider @Conduitry's suggestion?

**Comment by diverent2** [comment]
Hello there, since so much changed from the original PR I've opened a new one about the recommended changes here: 👉  https://github.com/sveltejs/sapper/pull/1199
See you there!

**Pull request closed without merging**